drv/transceivers-server/src/main.rs

```diff
@@ -688,6 +665,43 @@ fn main() -> ! {
             for t in multitimer.iter_fired() {
                 match t {
                     Timers::I2C => {
+                        // Handle the Front IO status checking as part of this
+                        // loop because the frequency is what we had before and
+                        // the server itself has no knowledge of the sequencer.
+                        if server.check_front_io_status {
```

Comment:
And with the `Option` above, this would change into something like:
```rust
if server.front_io_board_present.is_none() {
    server.front_io_board_present = match seq.front_io_board_ready() {
        Ok(true) => {
            ...

            Some(true)
        },
        Err(SeqError::NoFrontIOBoard) => {
            ...

            Some(false)
        },
        _ => {
            ringbuf_entry!(Trace::FrontIOBoardReady(
                false
            ));

            None
        }
    }
}
```

And then in the handler which responds to the network, it would become something like:
```rust
fn handle_network(&self, request) {
    if !self.front_io_board_present.or_default(false) {
        return Err(...)
    }
    
    ... handle request 
}
```

We could even be more specific to `dpd` if we want to:
```rust
fn handle_network(&self, request) {
    match self.front_io_board_present {
        None => return Err(Error::BoardNotReady)
        Some(false) => return Err(Error::BoardNotPresent)
        Some(true) => {}
    }
    
    ... handle request
}
```